### PR TITLE
Clarify 'low-contrast' in docstring

### PR DESCRIPTION
**Pull request by jni**
Just a small update to ensure the `fraction_threshold` parameter is clear to new users.

### CODE REVIEW

File: skimage/exposure/exposure.py [hunk 1]
```diff
@@ -499,6 +501,10 @@ def is_low_contrast(image, fraction_threshold=0.05, lower_percentile=1,
     True
     >>> is_low_contrast(image, upper_percentile=100)
     False
+
+    See Also
```
Comment (blink1073) [inline skimage/exposure/exposure.py]: See also is reserved by sphinx for referring to another place in the *code*.  You'll have to make this a `References` section.

**Comment by jni** [comment]
@blink1073 thanks for that! I'll update to References.

**Comment by jni** [comment]
Anyone got any ideas what's up with sphinx now?

**Comment by blink1073** [comment]
Oh great, Sphinx release a new version today...

**Comment by blink1073** [comment]
If you rebase after #1424 is merged you should be good to go.

**Comment by JDWarner** [comment]
#1424 is now merged.

**Comment by jni** [comment]
Done! Thanks for that!

**Comment by blink1073** [comment]
Well that was a lot of work for a simple doc change  :smile:

**Pull request merged**